### ISSUE TITLE
NiceHash Dashboard showing Too Many Digits

### ISSUE THREAD

**Issue by ihateyuu** [nicehash/NiceHashMiner-Archived#1309]
Hi All,

Is anyone getting same issue just like me

im getting too many digits on Estimated balance and my Balance

thank you

**Comment by dfair98** [comment]
having the same issue
![image](https://user-images.githubusercontent.com/4811749/31853355-dd6d348c-b63b-11e7-8c66-865ac0d85ffa.png)

**Comment by bradtrebilcock** [comment]
Yeah, seems they are updating the API. Hopefully another update will correct it.

**Comment by bradtrebilcock** [comment]
Weird I have another machine, same version and algo without this issue.

**Comment by dfair98** [comment]
now if i stop the miner, tada back to normal
![image](https://user-images.githubusercontent.com/4811749/31853442-105ed386-b63d-11e7-9b2c-7cf8e9ba843a.png)

**Comment by bradtrebilcock** [comment]
I shutdown NiceHash app and restarted and its fixed here

**Comment by dfair98** [comment]
@bradtrebilcock  i didn't even have to restart the app.

**Comment by bradtrebilcock** [comment]
@dfair98 I tried that first, but only after the app restart did it work for me. I thought I was going to get to retire... lol  all good now.

**Comment by jpmckearin** [comment]
App restart fixed it for me as well.

**Comment by sleeper252** [comment]
It's doing this on the nicehash website for me now.

**Comment by sleeper252** [comment]
It's doing this on the nicehash website for me right now...
![image](https://user-images.githubusercontent.com/32986068/31854023-b482d9da-b660-11e7-990d-31580825eca3.png)

**Comment by bradtrebilcock** [comment]
my mobile app as well, this has to be coming from the source/API. Are changes being made constantly throughout the day on this platform or are they scheduled?

**Comment by bradtrebilcock** [comment]
Also both miners Legacy and NiceHash 2 now have this problem

**Comment by bradtrebilcock** [comment]
Apps fixed again by restarting again. The website appears to be corrected already too. I'm sure someone is looking into it though.

**Comment by jpmckearin** [comment]
@ihateyuu I believe this can be closed now.
